samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/CommonMessages_ca.properties

```diff
@@ -55,3 +55,4 @@ cancel_question_info_no_undo=Aquesta acci\u00f3 no es pot desfer!
 cancel_question_info_skip_question=Podeu ometre aquesta pregunta.
 cancel_question_error_cancelling=S\u0027ha produït un error intentant cancel\u00b7lar la pregunta. No s\u0027han fet canvis.
 cancel_question_info_emi=Les preguntes d\u0027elements de correspond\u00e8ncia no rebr\u00e0 puntuacions ajustades.
+cancel_question_info_random=Questions of assessments with questions that are randomly assigned from question pools can not be cancelled.
```

Comment:
@DavidVegaSa, could you help with this translation, please?

Reply:
cancel_question_info_random=Les preguntes d\u0027avaluaci\u00f3 amb preguntes assignades aleat\u00f2riament dels grups de preguntes no es poden cancel\u00b7lar.

Reply:
Thanks @DavidVegaSa!